### PR TITLE
Further nuget sonar fixes

### PR DESCRIPTION
**Pull request by mrprescott**
Just warming up, knocking off a few 'critical' sonar issues. (I've subscribed to Sonar's 'new issues' emails, but haven't received one yet. Someone do some damage.)

### CODE REVIEW

**Comment by mrprescott** [comment]
http://bamboo.s/browse/NX3-OSSF425-2 :white_check_mark:

File: plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/CountReportingPolicy.java [hunk 1]
```diff
@@ -19,10 +19,14 @@
 import static java.util.Arrays.asList;
 
 /**
+ * Synthesizes a count from remote and hosted component counts.
+ *
  * @since 3.0
  */
 public class CountReportingPolicy
 {
+  private CountReportingPolicy() {}
```
Comment (cstamas) [inline plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/CountReportingPolicy.java]: In maven plugin this raised sonar "codestyle" warning, and required to put closing bracket on next line...

**Comment by cstamas** [comment]
+1

**Comment by bradbeck** [comment]
+1

File: plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/CountReportingPolicy.java (hunk 1 above)
Comment (jdillon) [inline plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/CountReportingPolicy.java]: sonar doesn't know our codestyle (atm) so you can/should ignore those

Comment (mrprescott) [inline plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/CountReportingPolicy.java]: @cstamas When this comes up next, we should neutralize the rule in Sonar.

**Comment by mrprescott** [comment]
Manually merged as 4a1ccb6

**Pull request closed without merging**